src/Provider/AbstractProvider.php

```diff
@@ -674,6 +674,15 @@ protected function parseResponse(ResponseInterface $response)
                 throw $e;
             }
 
+            if ($response->getStatusCode() == 500) {
+                throw new UnexpectedValueException(
+                    'Failed to parse JSON and OAuth server response code is
+ 500. It could be internal error on Oauth server.',
```

Comment:
This line break in the middle of the string is awkward. If trying to avoid going over 120 characters, please split the line up across lines using the concatenation operator.